docs/examples/imagestreams/is.yml

```diff
@@ -0,0 +1,9 @@
+name: webapp
+imageStreams:
+- tags:
+  - from:
+      kind: DockerImage
+      name: centos/httpd-24-centos7:latest
+    name: "2.4"
+    referencePolicy:
+      type: Source
```

Comment:
@containscafeine in the docs, specifically `is.yaml` can you show how the imagestream helps with deploymentconfig? Because just creating imagestream won't help to visualize what actually happened. With all other examples [[1](https://github.com/kedgeproject/kedge/blob/c64b0fd7a69edc4db5ef9aab0c52c97a0c9cf10e/docs/examples/includeResources/app.yaml#L21)] [[2](https://github.com/kedgeproject/kedge/blob/c64b0fd7a69edc4db5ef9aab0c52c97a0c9cf10e/docs/examples/customVol/db.yaml#L20)] [[3](https://github.com/kedgeproject/kedge/blob/c64b0fd7a69edc4db5ef9aab0c52c97a0c9cf10e/docs/examples/routes/httpd.yml#L9)] we are showcasing how that particular feature is helpful with the running application.

I know `is.yaml` is valid and will be converted and deployed without errors, still for someone who is new cannot visualize how she can use it with application.